src/ResourceGraph/ResourceGraph/Models/PSResourceGraphResponse.cs

```diff
@@ -14,17 +14,49 @@
 
 namespace Microsoft.Azure.Commands.ResourceGraph.Models
 {
+    using System.Collections;
     using System.Collections.Generic;
-    using System.Management.Automation;
     using Microsoft.WindowsAzure.Commands.Common.Attributes;
 
-    public class PSResourceGraphResponse
+    public class PSResourceGraphResponse<PSObject> : IList<PSObject>
     {
         [Ps1Xml(Target = ViewControl.List)]
```

Comment:
>[Ps1Xml(Target = ViewControl.List)] [](http://example.com/codeflow?start=8&length=35)

are these Ps1xml attributes with List control still accurate and applies?
and any ps1xml file changes needed?

Reply:
Good point for the ps1xml change, thank you! Updated it.
Technically List keyword is not changing anything anymore. Though I still left the List attribute since the default value is Table and List makes more sense to me for output of data and skip token.